packages/utils/test-tools/package.json

```diff
@@ -23,7 +23,8 @@
     "directory": "packages/utils/test-tools"
   },
   "dependencies": {
-    "@fluentui-react-native/theme-types": ">=0.14.3 <1.0.0"
+    "@fluentui-react-native/theme-types": ">=0.14.1 <1.0.0",
+    "@fluentui-react-native/theme-tokens": ">=0.16.1 <1.0.0"
```

Comment:
>@fluentui-react-native/theme-tokens" [](http://example.com/codeflow?start=5&length=36)

I'm a bit surprised that this package needs theme-tokens,  esp since it seems like there' no other changes in here. What's it used for?

Reply:
Oh my bad. I think I added this when trying to define the default variants for the mockTheme as well but didn't make those changes after all. Removing!